core/src/apps/webauthn/fido2.py

```diff
@@ -251,43 +249,32 @@ def frame_cont() -> uctypes.StructDict:
     }
 
 
-def resp_cmd_init() -> uctypes.StructDict:
-    # uint8_t nonce[8];         // Client application nonce
-    # uint32_t cid;             // Channel identifier
-    # uint8_t versionInterface; // Interface version
-    # uint8_t versionMajor;     // Major version number
-    # uint8_t versionMinor;     // Minor version number
-    # uint8_t versionBuild;     // Build version number
-    # uint8_t capFlags;         // Capabilities flags
```

Comment:
please return these comments (here and below)

Reply:
Done in [2f64875](https://github.com/trezor/trezor-firmware/pull/2525/commits/2f64875fd31a9f1ad8993bab35363c2ec0df3db4)